source/material_model/interface.cc

```diff
@@ -407,6 +411,25 @@ namespace aspect
 
 
 
+    template <int dim>
+    bool
+    MaterialModelInputs<dim>::requests_property(const MaterialProperties::Property &property) const
+    {
+      //TODO: Remove this once all callers set requested_properties correctly
+      if ((property & MaterialProperties::Property::viscosity) != 0)
+        return (strain_rate.size() != 0);
+
+      //TODO: Remove this once all callers set requested_properties correctly
+      if ((property & MaterialProperties::Property::reaction_terms) != 0)
+        return (strain_rate.size() != 0);
+
+      // Note that this means requested_properties can have other properties than
+      // property, but at least property.
```

Comment:
was this the intended wording?
```suggestion
      // property, but at least one property.
```